### PR TITLE
add index page

### LINKED ISSUE #24: add index page

### PR DESCRIPTION
**Pull request by mewiemers**
Fixes #24

Deployment: https://meet-albert-mewiemers-i-wgytmy.herokuapp.com/

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-import Card from "../components/card/Card";
+import Logo from "../components/welcome/Welcome";
 
 export default function Home() {
   return (
-    <div className={styles.container}>
+    <div>
       <Head>
-        <title>Create Next App</title>
-        <link rel="icon" href="/favicon.ico" />
```
Comment (RonjaMeinecke) [inline pages/index.tsx:10]: maybe consider to add Albert as a favicon next 😍

File: pages/index.tsx [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
+        <link rel="icon" href="/" />
       </Head>
 
-      <main className={styles.main}>
-        <div>
-          <Card primary label="Primary" />
-        </div>
+      <main>
+        <Logo></Logo>
+        <div></div>
```
Comment (RonjaMeinecke) [inline pages/index.tsx:14]:
```suggestion
        <div></div>
```
I think you don't need this placeholder at the moment

**Review by RonjaMeinecke — APPROVED** [review]
LGTM 🦘

File: pages/index.tsx [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
       </main>
 
-      <footer className={styles.footer}>Powered by Melanie</footer>
+      <footer></footer>
```
Comment (cboenke) [inline pages/index.tsx:17]: GJ, but no need for this footer :) 

```suggestion
      
```

**Review by cboenke — APPROVED** [review]

**Pull request merged**